frontend/integration-tests/views/crd.view.ts

```diff
@@ -0,0 +1,16 @@
+import { $, browser, by, element, ExpectedConditions as until } from 'protractor';
+
+export const crdItemsList = $('.co-m-resource-customresourcedefinition');
+
+export const clickOneCrd = async function(crdName: string, crdLink: string) {
```

Comment:
Per console naming conventions, CRD should be all caps

```suggestion
export const clickOneCRD = async function(crdName: string, crdLink: string) {
```